mlflow/langchain/__init__.py

```diff
@@ -641,7 +641,7 @@ def predict(
         if is_in_databricks_model_serving_environment() and MLFLOW_ENABLE_TRACE_IN_SERVING.get():
             from mlflow.langchain.langchain_tracer import MlflowLangchainTracer
 
-            callbacks = [MlflowLangchainTracer()]
+            callbacks = [MlflowLangchainTracer(prediction_context=get_prediction_context())]
```

Comment:
This change is necessary for propagating request_id into processor in serving. But we need to set prediction_context in serving